src/main/java/com/microsoft/azure/vmagent/AzureVMCloudVerificationTask.java

```diff
@@ -132,7 +133,7 @@ private static void verifyCloud(String cloudName) {
                         "AzureVMCloudVerificationTask: verify: cloud {0} already verified pass",
                         cloudName);
                 // Update the count.
-                cloud.setVirtualMachineCount(getVirtualMachineCount(cloud));
+                updateCloudVirtualMachineCounts(cloud);
```

Comment:
I'm not certain if this code is meant to be looking for/handling errors or not.
It looked like the old code would log errors and then simply return the previously-cached values making this a no-op in error situations.
I've made my new code return a boolean true on success and false on failure so _now_ this code could react to an error and do something about it if it wanted to.
...but I don't know if it should.